README.rst

```diff
@@ -24,19 +24,19 @@ On Unix, Linux, BSD, OSX, and Cygwin:
 
 This will install Python as python3.
 
-You can pass many options to the configure script; run "./configure --help" to
-find out more.  On OSX and Cygwin, the executable is called python.exe;
-elsewhere it's just python.
+You can pass many options to the configure script; run ``./configure --help`` to
+find out more.  On OSX and Cygwin, the executable is called ``python.exe``;
+elsewhere it's just ``python``.
 
-On Mac OS X, if you have configured Python with --enable-framework, you should
-use "make frameworkinstall" to do the installation.  Note that this installs the
+On Mac OS X, if you have configured Python with ``--enable-framework``, you should
+use ``make frameworkinstall`` to do the installation.  Note that this installs the
 Python executable in a place that is not normally on your PATH, you may want to
-set up a symlink in /usr/local/bin.
+set up a symlink in ``/usr/local/bin``.
 
 On Windows, see PCbuild/readme.txt.
```

Comment:
Such change are out of the scope of my initial change. I suggest to make more enhancements in a second change.